src/index.html

```diff
@@ -17,11 +17,696 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="/src/styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
```

Comment:
The class 'is-active' is used here, but on line 69, the class 'active' is used. Ensure consistency in class naming for active states across your navigation links.